core/src/main/scala/caliban/schema/Annotations.scala

```diff
@@ -81,4 +82,16 @@ object Annotations extends AnnotationsVersionSpecific {
    */
   case class GQLOneOfInput() extends StaticAnnotation
 
+  /**
+   * Compile-time annotation that can be used in conjunction with [[caliban.transformers.Transformer]] to
+   * customize schema generation.
+   */
+  case class GQLTag(tags: String*)
```

Comment:
We have an existing `GQLTag` annotation in federation since it's part of that spec already. Could we just move that instead?